Modules/Managers/PathManager.m

```diff
@@ -159,30 +141,25 @@ function select_path(obj,name,prompt,update_active)
             end
             
             path = obj.paths(map);
-            if ischar(path.instructions) % Alias
+            if isequal(path.name,obj.active_path); return; end
```

Comment:
important bug for manual paths - before, it would ask every time even if already on that path.